client/src/components/PastInterviews.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import Rating from '@material-ui/lab/Rating';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles( (theme) => ({
+  table: {
+    minWidth: 650,
+  },
+  btn: {
+    marginLeft: theme.spacing(1),
+    borderRadius: '30px'
+  },
+}));
+
+const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+
+const StyledTableCell = withStyles((theme) => ({
+    head: {
+      backgroundColor: theme.palette.primary.main,
+      color: theme.palette.common.white,
+      fontSize: 16,
+      fontWeight: '600'
+    },
+    body: {
+      fontSize: 14,
+      fontWeight: '400'
+    },
+}))(TableCell);
+  
+
+export default function UpcomingInterviews({ rows }) {
+  const classes = useStyles();
+
+  const formatDate = (date) => {
+    return `${days[date.getDay()]}, ${months[date.getMonth()]} ${date.getDate()} ${date.getFullYear()}`
+  }
+
+  const formatHour = (date) => {
+      const hours = date.getHours();
+      const minutes = date.getMinutes();
+      return `${hours < 12 ? hours : hours - 12}:${minutes}${minutes < 10 ? 0 : ''} ${hours < 13 ? 'AM': 'PM'}`
+  }
+
+  if (!rows || rows.length === 0){
+    return(
+      <Typography component="div" variant="h6">
+          <Box 
+              mt={3}
+              display="flex"
+              justifyContent="center"
+              alignItems="center" 
+              color="text.disabled"
+          >
+              There are no past interviews.
+          </Box>
+      </Typography>
+    )
+}
+
+  return (
+    <Box mt={3}>
```

Comment:
In general I recommend using `<Grid>` instead of `<Box>`. Grid is a wrapper for box and includes more features. The only reason to use Box is if for some reason you want to get really granular with styling. Otherwise go with Grid!

Reply:
I think in this case Box should be okay, since there's no flexbox in use here.